pkgs/by-name/in/inhibridge/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitea,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "inhibridge";
+  version = "0.3.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "Scrumplex";
+    repo = "inhibridge";
+    rev = version;
```

Comment:
use `finalAttrs` instead of `rec`
```suggestion
    rev = finalAttrs.version;
```

Reply:
IIRC `rustPlatform.buildRustPackage` doesn't support function based attributes